sentry.sublime-project

```diff
@@ -0,0 +1,44 @@
+{
+    "folders":
+    [
+        {
+            "path": ".",
+        }
+    ],
+    "settings":
+    {
+        "tab_size": 4,
```

Comment:
its just the default, but yeah good point if it overrides langauge settings that'd be a problem